api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -143,7 +144,17 @@ private void voidOrders(Collection<Patient> patientsWithOrders) {
 			}
 		}
 	}
-	
+
+	private void voidOrders(Collection<Patient> patients, OrderType ot) {
```

Comment:
Something like voidOrdersForType or anything else along those lines would make this method more meaningful.